### ISSUE TITLE
4K 240Hz HDMI issue (Screen going black for ~2 second)

### ISSUE THREAD

**Issue by Denizeri24** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#767]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game [Required]

None

### Game Platform [Required]

- [ ] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [ ] Amazon Games app
- [X] Other (please specify)

### Other game platform

None

### Processor / Processor Number [Required]

i7 8700K

### Graphic Card [Required]

A770 16 GB

### GPU Driver Version [Required]

31.0.101.5382

### Rendering API [Required]

- [X] Vulkan
- [X] OpenGL
- [X] DirectX12
- [X] DirectX11
- [X] DirectX10
- [X] DirectX9

### Windows Build Number [Required]

- [X] Windows 11 23H2
- [ ] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

--Not needed--
[scanoutput.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/15039669/scanoutput.txt)

### Description and steps to reproduce [Required]

I just using my monitor (MSI 321 URX -- 4K / 240Hz) with HDMI 2.1 cable and screen constantly going black (~2 secs) for every ~1 min. I dont have this issue if I use with DP 1.4 cable.

Also if I enable vsync with DP 1.4 cable, my screen going black and my pc doing infinite restart (probably gpu crash) (I have this problem if i using DP 1.4 cable).

### Game graphic quality [Required]

- [X] Low
- [X] Medium
- [X] High
- [X] Ultra

### Game resolution [Required]

4K

### Game VSync [Required]

On

### Game display mode [Required]

- [X] Fullscreen
- [X] Windowed
- [X] Borderless Windowed

### Detailed game settings [Required]

Not needed

**Comment by EstebanIntel** [comment]
Hi @Denizeri24,

As stated here: Intel® [Arc™ A770 Graphics ](https://ark.intel.com/content/www/us/en/ark/products/229151/intel-arc-a770-graphics-16gb.html), the max resolution supported by Arc over HDMI is 4K@60Hz. While the max resolution supported over DP is 8K@60Hz. This means that 4K@240Hz is expected to work over DP, but not over HDMI.

**Comment by EstebanIntel** [comment]
Hi @Denizeri24,

There are bandwidth limitations on both the GPU and Monitor's sides for the high resolution and refresh rate you are trying to use. Let's start with the Arc GPU:

As stated here: Intel® [Arc™ A770 Graphics ](https://ark.intel.com/content/www/us/en/ark/products/229151/intel-arc-a770-graphics-16gb.html), the max resolution supported by Arc over HDMI 2.1 is 4K@60Hz. While the max resolution supported over DP 2.0 is 8K@60Hz.

Now, on the monitor's side:

I see you are using the MSI 321URX. From its specs, it features DP1.4a (HBR3) and HDMI 2.1 interfaces. From the [Wikipedia article on DP](https://en.wikipedia.org/wiki/DisplayPort), from table [Refresh frequency limits for standard video](https://en.wikipedia.org/wiki/DisplayPort#Refresh_frequency_limits_for_standard_video) you can see that the max DP1.4a (HBR3) can do is 4K@120Hz for SDR; and from table [Refresh frequency limits for HDR video](https://en.wikipedia.org/wiki/DisplayPort#Refresh_frequency_limits_for_HDR_video) the max for HDR is 4K@60Hz. 
On the HDMI side, the [Wikipedia article for HDMI](https://en.wikipedia.org/wiki/HDMI), from table [Refresh frequency limits for standard video](https://en.wikipedia.org/wiki/HDMI#Refresh_frequency_limits_for_standard_video) you can see that the limit for HDMI 2.1 on SDR is 4K@144Hz (since I don't think your monitor support DSC, but I might be wrong); and from the table [Refresh frequency limits for HDR10](https://en.wikipedia.org/wiki/HDMI#Refresh_frequency_limits_for_HDR10_video) the max on HDR is 4K@60.

In summary, the max supported resolution/refresh rate for your GPU and Monitor combination is what I'm listing in column "Combined Limit" in the table below. 

Interface | Dynamic Range | Arc A-series Limit | MSI 321URX Limit | Combined Limit
-- | -- | -- | -- | --
HDMI | SDR | 4K@60 | 4K@144 | 4K@60
HDMI | HDR | 4K@60 | 4K@60 | 4K@60
DP | SDR | 8K@60 | 4K@120 | 4K@120
DP | HDR | 8K@60 | 4K@60 | 4K@60

**Comment by Denizeri24** [comment]
> Hi @Denizeri24,
> 
> There are bandwidth limitations on both the GPU and Monitor's sides for the high resolution and refresh rate you are trying to use. Let's start with the Arc GPU:
> 
> As stated here: Intel® [Arc™ A770 Graphics ](https://ark.intel.com/content/www/us/en/ark/products/229151/intel-arc-a770-graphics-16gb.html), the max resolution supported by Arc over HDMI 2.1 is 4K@60Hz. While the max resolution supported over DP 2.0 is 8K@60Hz.
> 
> Now, on the monitor's side:
> 
> I see you are using the MSI 321URX. From its specs, it features DP1.4a (HBR3) and HDMI 2.1 interfaces. From the [Wikipedia article on DP](https://en.wikipedia.org/wiki/DisplayPort), from table [Refresh frequency limits for standard video](https://en.wikipedia.org/wiki/DisplayPort#Refresh_frequency_limits_for_standard_video) you can see that the max DP1.4a (HBR3) can do is 4K@120Hz for SDR; and from table [Refresh frequency limits for HDR video](https://en.wikipedia.org/wiki/DisplayPort#Refresh_frequency_limits_for_HDR_video) the max for HDR is 4K@60Hz. On the HDMI side, the [Wikipedia article for HDMI](https://en.wikipedia.org/wiki/HDMI), from table [Refresh frequency limits for standard video](https://en.wikipedia.org/wiki/HDMI#Refresh_frequency_limits_for_standard_video) you can see that the limit for HDMI 2.1 on SDR is 4K@144Hz (since I don't think your monitor support DSC, but I might be wrong); and from the table [Refresh frequency limits for HDR10](https://en.wikipedia.org/wiki/HDMI#Refresh_frequency_limits_for_HDR10_video) the max on HDR is 4K@60.
> 
> In summary, the max supported resolution/refresh rate for your GPU and Monitor combination is what I'm listing in column "Combined Limit" in the table below.
> 
> Interface	Dynamic Range	Arc A-series Limit	MSI 321URX Limit	Combined Limit
> HDMI	SDR	4K@60	4K@144	4K@60
> HDMI	HDR	4K@60	4K@60	4K@60
> DP	SDR	8K@60	4K@120	4K@120
> DP	HDR	8K@60	4K@60	4K@60

Shouldn't HDMI 2.1 support 4K + 240Hz resolution with DSC? My monitor supports DSC..

**Comment by Denizeri24** [comment]
> @Denizeri24 thanks for confirming your monitor does support DSC. Please find below the updated table with this info. As you can see, for HDMI SDR your monitor does support 4K@240Hz. However, the limit is on the Arc GPU side, as [Arc only support up to 4K@60Hz over HDMI](https://ark.intel.com/content/www/us/en/ark/products/229151/intel-arc-a770-graphics-16gb.html#tab-blade-1-0-4).
> 
> Interface	Dynamic Range	Arc A-series Limit	MSI 321URX Limit	Combined Limit
> HDMI	SDR	4K@60	4K@240	4K@60
> HDMI	HDR	4K@60	4K@60	4K@60
> DP	SDR	8K@60	4K@120	4K@120
> DP	HDR	8K@60	4K@60	4K@60

why arc have a limit like 4k 60 hz? i read some articles about arc and they saying arc supports hdmi 2.1 natively. also if combined limit is 4k 120 hz for DP, how can i get 4k / 240 hz with DP output?

**Comment by EstebanIntel** [comment]
@Denizeri24, the HDMI port on Arc A-Series in not native HDMI, but a DP port converted to HDMI using a PCON. This results in the limitation mentioned in the A770 specs for HDMI max resolution/refresh rate.

In general, the Arc A-Series is not capable is displaying 4K@240Hz, as this is beyond the performance tier of this GPU. This GPU is intended to be used for 1080p/1440p gaming. And although this GPU might be capable of running some games at 4K, it will do it at lower framerates; and it's unlikely this GPU will be able to achieve 240 fps @ 4K.

If you are looking to set your refresh-rate at 240Hz, your best bet is to lower the resolution to 1440p.

**Comment by EstebanIntel** [comment]
Hi @Denizeri24, do you have any further question? If not, can this be closed?

**Comment by Dredd893** [comment]
What a great response by you EstebanIntel. Great support, and the reason I will be purchasing Battlemage when it will be out.

The guys in the higher sphere should make a shrine for the people that work on driver and on hardware for GPU.

Tier S for your behaviour. Well done.

**Comment by Denizeri24** [comment]
> the HDMI port on Arc A-Series in not native HDMI, but a DP port converted to HDMI using a PCON. This results in the limitation mentioned in the A770 specs for HDMI max resolution/refresh rate.
> 
> In general, the Arc A-Series is not capable is displaying 4K@240Hz, as this is beyond the performance tier of this GPU. This GPU is intended to be used for 1080p/1440p gaming. And although this GPU might be capable of running some games at 4K, it will do it at lower framerates; and it's unlikely this GPU will be able to achieve 240 fps @ 4K.
> 
> If you are looking to set your refresh-rate at 240Hz, your best bet is to lower the resolution to 1440p.

Ah, I dont get an e-mail for your response, sorry for that. you can close this thread, thanks!
